docs/UsersGuide/source/Graphics.rst

```diff
@@ -161,6 +169,8 @@ Cheyenne that use PBS or PBS Pro as the job scheduler).  These scripts are locat
 for the job scheduler used on the current platform.  For example, on Hera, Jet, Orion, and Gaea, 
 ``sq_job.sh`` can be submitted as follows:
 
+NOTE DOM 20220227  - this script will *not* work on gaea for sure. First of all because the module load commands are wrong (see above, second because gaea uses #SBATCH --clusters, third because you cannot do module purge and module load hpss). The line ". /apps/lmod/lmod/init/sh" doesn't work on gaea, it doesn't work on jet either.
+
 .. code-block:: console
```

Comment:
Given the major differences between Gaea and Jet/Hera/Orion, and the current time crunch, I would suggest we list the sbatch script as not currently functional on Gaea in the known limitations document.

Reply:
I changed the wording to the following: (Thoughts? Comments?)
If the Python scripts are being used to create plots of multiple forecast lead times and forecast
variables, then they should be submitted to the batch system using either something like the ``sq_job.sh``
or ``sq_job_diff.sh`` script (for a platform such as Hera that uses the slurm
job scheduler) or the ``qsub_job.sh`` or ``qsub_job_diff.sh`` script (for a platform such as
Cheyenne that uses PBS or PBS Pro as the job scheduler).  Examples of these scripts are located under
``ufs-srweather-app/regional_workflow/ush/Python`` and can be used as a starting point to create a batch script
for your platform/job scheduler of use. The scripts must be submitted using the command appropriate
for the job scheduler used on your platform.  For example, on Hera,
``sq_job.sh`` can be submitted as follows:

.. code-block:: console

   sbatch sq_job.sh

On Cheyenne, ``qsub_job.sh`` can be submitted as follows:

.. code-block:: console

   qsub qsub_job.sh